package.json

```diff
@@ -5,7 +5,7 @@
   "private": true,
   "main": "app.js",
   "scripts": {
-    "test": "make test",
+    "test": "snyk test && make test",
```

Comment:
Out of interest why `snyk test` before `make test`? 

Just curious as if I'm working on something + running tests I'd probably want to the deps to stay stable whilst I'm doing so. But this may well just be me. I guess we so similar things with `make verify` elsewhere too?

Reply:
No idea! This is just what it adds itself.

Reply:
:magic: